### DISCUSSION TITLE
File renaming not working?

### DISCUSSION THREAD

**Discussion by dummythicc99** [DIGITALCRIMINALS/OnlyFans#224] [Q&A]
The files I scrape don't appear to be getting renamed to what I specify. Here's what the config looks like:

"onlyfans": {
      "settings": {
        "auto_profile_choice": [],
        "auto_model_choice": false,
        "auto_media_choice": "",
        "auto_api_choice": true,
        "browser": {
          "auth": true
        },
        "jobs": {
          "scrape_names": true,
          "scrape_paid_content": true
        },
        "download_directories": [
          ".sites"
        ],
        "file_directory_format": "{site_name}\\{model_username}\\{api_type}\\{value}\\{media_type}",
        **_"filename_format": "{date}.{ext}",_**
        "metadata_directories": [
          ".sites"
        ],
        "metadata_directory_format": "{site_name}\\{model_username}\\Metadata",
        "delete_legacy_metadata": false,
        "text_length": 255,
        "video_quality": "source",
        "overwrite_files": false,
        "date_format": "%d-%m-%Y",
        "ignored_keywords": [],
        "ignore_type": "",
        "blacklists": "",
        "webhook": true

Yet the filenames appear as, for example "5f75275e1da71ff5e57e4_source.mp4" for videos and "945x1681_4506a701fbd810ff5b02b329f882fe33.jpg" for pics. Anyone know if something's up?

**Comment by DIGITALCRIMINAL** [comment]
Your filename is invalid, it will overwrite multiple content with the same date
